src/styles/theme/default.theme.ts

```diff
@@ -52,6 +52,7 @@ export const oakDefaultTheme: OakTheme = {
     "border-primary": "black",
     "border-inverted": "white",
     "border-neutral": "grey50",
+    "border-neutral-lighter": "grey40",
```

Comment:
Looks like this has been added to the design kit.

https://www.figma.com/file/YcWQMMhHPVVmc47cHHEEAl/Oak-Design-Kit?type=design&node-id=6113-14781&mode=dev